src/pages/TrueSightV2/pages/News.tsx

```diff
@@ -0,0 +1,185 @@
+import { Text } from 'rebass'
```

Comment:
Is this page being used 😄 It seems that those news datas are hardcoded.

Reply:
yes removed